crates/holochain/src/core/workflow/initialize_zomes_workflow.rs

```diff
@@ -69,6 +72,9 @@ where
             coordinators,
         )
         .await?;
+
+        // Any ops that were moved to the dht_db as part of the flush but had dependencies will need to be integrated.
+        integrate_dht_ops_trigger.trigger(&"initialize_zomes_workflow");
```

Comment:
Just to explain this change. The publish workflow used to trigger sys validation, which triggered app validation, which triggers integration. Those triggers are now conditional on work actually having been done by each workflow (at least for publish and sys validation). We need to explicitly trigger integration in a few places now. It wasn't great that we were relying on the validation workflows running to trigger integration and bugs following this change are entirely possible. There were broken tests so I hope I've caught all the cases but I'm not certain.

Reply:
What is the connection here between zome initializations and integration?

Reply:
And why did publish use to trigger sys validation?